content/de/how-it-works.md

```diff
@@ -8,57 +8,59 @@ lastmod: 2019-10-18
+Let's&nbsp;Encrypt identifiziert den Serveradministrator anhand des öffentlichen Schlüssels.  Wenn die Agentensoftware zum ersten Mal mit Let's&nbsp;Encrypt interagiert, generiert sie ein neues Schlüsselpaar und weist der Let's&nbsp;Encrypt-Zertifizierungsstelle nach, dass der Server eine oder mehrere Domain kontrolliert.  Dies ist vergleichbar mit dem traditionellen CA-Verfahren, bei dem ein Konto erstellt und diesem Konto Domains hinzugefügt werden.
 
-Um den Prozess zu starten, fragt der Agent die Let's Encrypt-Zertifizierungsstelle, was zu tun ist, um zu beweisen, dass sie `example.com` kontrolliert. Die Let's Encrypt-Zertifizierungsstelle prüft den angeforderten Domainnamen und gibt eine oder mehrere Herausforderungen heraus. Dies sind verschiedene Möglichkeiten, mit denen der Agent die Kontrolle über die Domain nachweisen kann. Beispielsweise kann die Zertifizierungsstelle dem Agenten die Wahl zwischen folgenden Optionen geben:
+Um den Prozess zu starten, fragt der Agent die Let's Encrypt-Zertifizierungsstelle, was zu tun ist, um zu beweisen, dass sie `example.com` kontrolliert.  Die Let's Encrypt-Zertifizierungsstelle prüft den angeforderten Domainnamen und gibt eine oder mehrere Herausforderungen heraus.   Dies sind verschiedene Möglichkeiten, mit denen der Agent die Kontrolle über die Domain nachweisen kann.  Beispielsweise kann die Zertifizierungsstelle dem Agenten die Wahl zwischen folgenden Optionen geben:
 
 * Bereitstellung eines DNS-Eintrags unter `example.com` oder
 * Bereitstellung einer HTTP-Ressource unter einem bekannten URI unter `http://example.com/`
 
 Neben den Herausforderungen bietet die Let's Encrypt-Zertifizierungsstelle auch eine Nonce, die der Agent mit seinem privaten Schlüsselpaar signieren muss, um zu beweisen, dass er das Schlüsselpaar kontrolliert.
 
-
 <div class="howitworks-figure">
 <img alt="Aufforderung zur Validierung von example.com stellen"
      src="/images/howitworks_challenge.png"/>
 </div>
 
-Die Agentensoftware erfüllt eine der gestellten Herausforderungen. Nehmen wir an, sie ist in der Lage, die zweite Aufgabe oben auszuführen: Sie erstellt eine Datei in einem angegebenen Pfad auf der Website `http://example.com`. Der Agent signiert die bereitgestellte Nonce ausserdem mit seinem privaten Schlüssel. Nachdem der Agent diese Schritte ausgeführt hat, benachrichtigt er die Zertifizierungsstelle, dass sie zur Validierung bereit ist.
+Die Agentensoftware erfüllt eine der gestellten Herausforderungen.   Nehmen wir an, sie ist in der Lage, die zweite Aufgabe oben auszuführen: Sie erstellt eine Datei in einem angegebenen Pfad auf der Website `http://example.com`.  Der Agent signiert die bereitgestellte Nonce außerdem mit seinem privaten Schlüssel.  Nachdem der Agent diese Schritte ausgeführt hat, benachrichtigt er die Zertifizierungsstelle, dass sie zur Validierung bereit ist.
```

Comment:
Die Agentensoftware erfüllt eine der gestellten Herausforderungen. Nehmen wir an, sie ist in der Lage, die zweite Aufgabe oben auszuführen: Sie erstellt eine Datei in einem angegebenen Pfad auf der Website `http://example.com`.  Der Agent signiert die bereitgestellte Nonce außerdem mit seinem privaten Schlüssel. Nachdem der Agent diese Schritte ausgeführt hat, benachrichtigt er die Zertifizierungsstelle, dass sie zur Validierung bereit ist.

Reply:
See above.